content/docs/installation/supported-releases.md

```diff
@@ -167,12 +166,30 @@ Kubernetes API or the command line flags. We avoid making breaking changes
 where possible, and where they're required we'll give as much notice as
 possible.
 
+<h4 id="other-backports">Other back-ports</h4>
+
+We aim to be extremely conservative in what we back-port. That applies especially for anything
+which could be a _runtime_ change - that is, a change which might alter behavior for someone
+upgrading between patch releases.
+
+That means that if a candidate for back-porting has a chance of having a runtime impact we're
+unlikely to accept the change unless it addresses a security issue or a critical bug.
+
+We reserve the right to back-port other changes which are unlikely to have a runtime impact, such as
+documentation or tooling changes. An example would be [#5209][] which updated how we perform a release of
+cert-manager but didn't have any realistic chance of having a runtime impact.
```

Comment:
I think it's great to write down/clarify all this 😄

Personally I am more conservative wrt making changes to the tooling surrounding making a release, as I think these _can_ affect runtime of cert-manager (e.g. if we make a change to release tooling that means say, the `acmesolver` image is not updated, then this could quietly break an end-users renewal).

I think there _is_ definitely room for some discretion here (as noted in #5209), but in general I think we should try and avoid any changes as much as possible (i.e. develop tooling changes/features & deprecation cycles in such a way that enables us to not have to modify old release branches too)

Reply:
I proposed the following rule of thumb which I think is a good mix of pragmatism and conservatism:

> "Does this back-port improve cert-manager, bearing in mind that we really value stability for already-released versions?"

What do you propose instead?

Reply:
Summarising it in one sentence, I'd say "The stability of already-released versions & their following patch versions is valued above all else. Only the most essential patches needed to ensure the user's security & reliability should be merged." (i.e. security fixes & regressions is an example).

> "Does this back-port improve cert-manager, bearing in mind that we really value stability for already-released versions?"

Regardless on what definition that we go for here, I actually think this statement right now _doesn't_ cover the rclone changes mentioned previously in the paragraph, as from an end-user perspective it has not been improved & it doesn't improve/increase the stability of the release either. (so I definitely think _something_ needs to change here :))

Reply:
To make absolutely clear here - I am not calling into question whether that rclone PR should have been backported. I am simply highlighting that we give that PR as a reference in this text, and then the rule of thumb at the bottom introduces ambiguity as to whether or not it is covered by it, as it seems definition of an "improvement" differs/it's quite a broad term.